utils/jsonnet.py

```diff
@@ -21,7 +21,7 @@ def generate_object(jsonnet_string):
     try:
         jsonnet_bundler_dir = os.environ['JSONNET_VENDOR_DIR']
     except KeyError as e:
-        raise JsonnetError('JSONNET_VENDOR_DIR not set')
+        raise JsonnetError(f'JSONNET_VENDOR_DIR not set: {e}')
```

Comment:
Is this to make code checker happy? you can remove the ` as e` instead... not need to throw extra info in the message if you're missing it :)

Reply:
I think it is worth matching the lines above so they are consistent

Reply:
let me represent @rporres and say that this is an unrelated change.

Reply:
The previous call was made out of `Exception`, which is as generic as you can get, so casting the original exception to a string makes sense as you don't really know what happened there.
Here you are handling `KeyError`, which is specific. You have a message saying exactly what's wrong, so there's no added value.... you're just making more noise.